mobile/src/bundles/common/components/loader/styles.ts

```diff
@@ -0,0 +1,12 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+    wrapper: {
+        width: '100%',
+        height: '100%',
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+});
```

Comment:
Whole style object can be replaced with global styles

Reply:
fix